layers/core_checks/device_memory_validation.cpp

```diff
@@ -374,31 +374,33 @@ bool CoreChecks::PreCallValidateFreeMemory(VkDevice device, VkDeviceMemory mem,
 //  and that the size of the map range should be:
 //  1. Not zero
 //  2. Within the size of the memory allocation
-bool CoreChecks::ValidateMapMemRange(const DEVICE_MEMORY_STATE *mem_info, VkDeviceSize offset, VkDeviceSize size) const {
+bool CoreChecks::ValidateMapMemRange(const DEVICE_MEMORY_STATE *mem_info, bool map2, VkDeviceSize offset, VkDeviceSize size) const {
     bool skip = false;
     assert(mem_info);
     const auto mem = mem_info->mem();
     if (size == 0) {
-        skip = LogError(mem, "VUID-vkMapMemory-size-00680", "VkMapMemory: Attempting to map memory range of size zero");
+        skip = LogError(mem, map2 ? "VUID-VkMemoryMapInfoKHR-size-07960" : "VUID-vkMapMemory-size-00680",
+                        "VkMapMemory: Attempting to map memory range of size zero");
```

Comment:
To avoid potential confusion perhaps best to not prefix the function?

```suggestion
                        "Attempting to map memory range of size zero");
```